File: CORREDOR/4 Lenguajes/C#/ProyectoCoche/Metodos_Parametros.cs
using System;

    class Metodos_Parametros
    {
        void Sumar(int a, int b)
        {
            int resultado = a + b;
            Console.WriteLine("La suma es: " + resultado);
        }

        static void Main(string[] args)
        {
            Metodos_Parametros cal = new Metodos_Parametros();
            cal.Sumar(8, 7);

            Resta r1 = new Resta();
            r1.Restar(18, 5);

            AreaTriangulo area = new AreaTriangulo();
            area.Area(8, 6);

            Promedio pro = new Promedio();
            pro.Inicial(9, 18, 12);
        }
    }

    class Resta
    {
        public void Restar(int a, int b)
        {
            int resultado = a - b;
            Console.WriteLine("La resta es: " + resultado);
        }
    }

    class AreaTriangulo
    {
        public void Area(int baseT, int altura)
        {
            int resultado = baseT * altura / 2;
            Console.WriteLine("El área del triángulo es: " + resultado);
        }
    }

    class Promedio
    {
        public void Inicial(int a, int b, int c)
        {
            int resultado = a + b + c;
            int promedio = resultado / 3;
            Console.WriteLine("El promedio es: " + promedio);
        }
}

File: CORREDOR/4 Lenguajes/Java/Metodos_Parametros.java
package Java;

public class Metodos_Parametros {
    void sumar(int a, int b){
        int resultado = a + b;
        System.out.println("La suma es: " + resultado);
    }

    public static void main(String[] args) {
        Metodos_Parametros cal = new Metodos_Parametros();
        cal.sumar(8, 7);

        Resta r1 = new Resta();
        r1.restar(18, 5);

        areaTriangulo area = new areaTriangulo();
        area.area(8, 6);

        Promedio pro = new Promedio();
        pro.inicial(9, 18, 12);
    }
}

    class Resta{
    void restar(int a, int b){
        int resultado = a - b;
        System.out.println("La resta es: " + resultado);
    }
}

    class areaTriangulo{
    void area(int base, int altura){
        int resultado = base * altura / 2;
        System.out.println("El area del triangulo: " + resultado);
    }
}

    class Promedio{
    void inicial(int a, int b, int c){
        int resultado = a + b + c;
        int promedio = resultado / 3;
        System.out.println("El promedio es: " + promedio);
    }
}